website/static/js/index.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-var */
 /* global katex: false */
-window.startup = function() {
+document.addEventListener("DOMContentLoaded", function() {
```

Comment:
This is to support browsers that don't support `defer`, but I'm not sure it's necessary.